.gitignore

```diff
@@ -3,6 +3,26 @@
 media
 db.sqlite3
 test_db.sqlite3
-
+__pycache__
 .tox
 .cache
+
+# exclude idea ide files
+.idea/
+next.filmfest.by.iml
+
+# exclude root json data files added by accedent
+/*.json
```

Comment:
s/exclude root json data files added by accedent/@davojta working snippets and notebooks/